internal/tests/cli/boundary/workers.bats

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bats
+
+load _auth
+load _workers
+load _helpers
+
+export NEW_WORKER='test'
+
+export NEW_UPDATED_WORKER='newtest'
+
+@test "boundary/login: can login as default user" {
+  run login $DEFAULT_LOGIN
+  echo "$output"
+  [ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: can create $NEW_WORKER worker" {
+	run create_worker $NEW_WORKER
+  echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: can not create already created $NEW_WORKER worker" {
+	run create_worker $NEW_WORKER
+  echo "$output"
+	[ "$status" -eq 1 ]
+}
+
+@test "boundary/workers: can read $NEW_WORKER worker" {
+  local wid=$(worker_id $NEW_WORKER)
+	run read_worker $wid
+  echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: can list workers" {
+	run list_workers
+  echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: the worker contains default authorized-actions" {
+  local wid=$(worker_id)
+  local out=$(read_worker $wid)
+	run has_default_worker_actions "$out"
+  echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: can update worker's name" {
+  local wid=$(worker_id $NEW_WORKER)
+    run update_worker $wid $NEW_UPDATED_WORKER
+  echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "boundary/workers: can delete $NEW_UPDATED_WORKER worker" {
+  local wid=$(worker_id $NEW_UPDATED_WORKER)
+  run delete_worker $wid
+  echo "$output"
+  run has_status_code "$output" "204"
+  [ "$status" -eq 0 ]
+}
```

Comment:
I've seen some other bats tests that check that it can't read the deleted object. Not sure what your thoughts are there. 

On the flip side, I *might* be seeing some flakiness in that kind of test. A theory is that there's some sort of timing thing where boundary hadn't finished completed deleting the entity before reading it again.